### PR TITLE
Lp/thomas / second test with Helmet. Layout stuff.

### PR DESCRIPTION
**Pull request by TomAlunni**

### CODE REVIEW

**Comment by TomAlunni** [comment]
Added AOS (Animate on Scroll) library to it

**Comment by Ryu-Sen** [comment]
> Added AOS (Animate on Scroll) library to it

does it? Animate on scroll?

**Comment by TomAlunni** [comment]
> > Added AOS (Animate on Scroll) library to it
> 
> does it? Animate on scroll?

Yes. It. Does.

File: src/components/index.js [hunk 1]
```diff
@@ -2,3 +2,4 @@ export { default as Header } from "./Header";
 export { default as Footer } from "./Footer";
 export { default as ListCarousel } from "./ListCarousel";
 export { default as Team } from "./Team";
+export { default as Layout } from "./Layout";
```
Comment (Ryu-Sen) [inline src/components/index.js]: cute

**Pull request merged**